quill-core/src/main/scala/io/getquill/norm/AttachToEntity.scala

```diff
@@ -5,13 +5,22 @@ import io.getquill.ast._
 
 object AttachToEntity {
 
-  def apply(f: (Query, Ident) => Query, alias: Option[Ident] = None)(q: Query): Query =
+  private object IsEntity {
+    def unapply(q: Ast): Option[Ast] =
+      q match {
+        case q: Entity => Some(q)
+        case q: Infix  => Some(q)
+        case _         => None
```

Comment:
How this could be related to other quill modules? I suppose these changes required to generate queries like this:
`SELECT x1.age _1 FROM (?) x1 WHERE x1.name = ?`. However does it impact other world somehow? e.g. adds new opportunities?

Reply:
I'd say this is a bug fix. Users should be able to use an `infix` in an entity position.